modules/client-configuration/pages/supported-features-es.adoc

```diff
@@ -0,0 +1,69 @@
+|===
+| Feature | {slsa}{nbsp}ES{nbsp}6 |  {slsa}{nbsp}ES{nbsp}7
+| Client | icon:check[role="green"] | icon:check[role="green"]
+| Operating system packages | icon:check[role="green"] | icon:check[role="green"]
+| Registration | icon:check[role="green"] | icon:check[role="green"]
+| Install packages | icon:check[role="green"] | icon:check[role="green"]
+| Apply patches | icon:check[role="green"] | icon:check[role="green"]
+| Remote commands | icon:check[role="green"] | icon:check[role="green"]
+| System package states | ``Salt`` | ``Salt``
+| System custom states | ``Salt`` | ``Salt``
+| Group custom states | ``Salt`` | ``Salt``
+| Organization custom states | ``Salt`` | ``Salt``
+| System set manager (SSM) | ``Salt`` | ``Salt``
+| Service pack migration | icon:times[role="danger"] | icon:times[role="danger"]
+| Basic Virtual Guest Management icon:asterisk | ``Traditional`` | icon:check[role="green"]
+| Advanced Virtual Guest Management icon:asterisk | icon:times[role="danger"] | ``Salt``
+| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional``
+| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"]
+| System redeployment ({ay}) | ``Traditional`` | icon:check[role="green"]
+| Contact methods | ``Traditional``: OSAD, RHNSD, SSH-push. +
+``Salt``: ZeroMQ, Salt-SSH | ``Traditional``: OSAD, RHNSD, SSH-push. +
+``Salt``: ZeroMQ, Salt-SSH
+| Works with {productname} Proxy | icon:check[role="green"] | icon:check[role="green"]
+| Action chains | icon:check[role="green"] | icon:check[role="green"]
+| Software crash reporting | icon:times[role="danger"] | ``Traditional``
+| Staging (pre-download of packages) | icon:check[role="green"] | icon:check[role="green"]
+| Duplicate package reporting | icon:check[role="green"] | icon:check[role="green"]
+| CVE auditing | icon:check[role="green"] | icon:check[role="green"]
+| SCAP auditing | icon:check[role="green"] | icon:check[role="green"]
+| Package verification | ``Traditional`` | ``Traditional``
+| Package locking | ``Traditional`` | ``Traditional``
+| System locking | ``Traditional`` | ``Traditional``
+| System snapshot | ``Traditional`` | ``Traditional``
+| Configuration file management | icon:check[role="green"] | icon:check[role="green"]
+| Snapshots and profiles |  ``Traditional``. +
+``Salt``: Profiles supported, Sync not supported | ``Traditional``. +
+``Salt``: Profiles supported, Sync not supported
+| Power management |  icon:check[role="green"] | icon:check[role="green"]
+| Monitoring |  ``Salt`` | ``Salt``
+| Docker buildhost |  icon:times[role="danger"] | icon:times[role="danger"]
+| Build Docker image with OS | icon:question[role="gray"] | icon:question[role="gray"]
+| Kiwi buildhost |  icon:times[role="danger"] | icon:times[role="danger"]
+| Build Kiwi image with OS |  icon:times[role="danger"] | icon:times[role="danger"]
+|===
+
+icon:asterisk Virtual Guest Management:
```

Comment:
This doesn't render.

Reply:
In my PR I used ``icon:asterisk[role="none"]``. Not sure why they're all removed now.